assets/airports/airportLoadList.js

```diff
@@ -235,8 +235,7 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             icao: 'khou',
             level: 'easy',
```

Comment:
Current total acph is 13.68 which would fall under beginner level. If this value is still under 20 after the addition of GA spawnPatterns, the level should be changed to beginner.